monkestation/code/modules/power/rbmk/rbmk_meltdown_fallout.dm

```diff
@@ -0,0 +1,114 @@
+/obj/effect/decal/nuclear_waste
+	name = "Plutonium sludge"
+	desc = "A writhing pool of heavily irradiated, spent reactor fuel. You probably shouldn't step through this..."
```

Comment:
Have it say to use a shovel to clear it.
I serve the Space Soviet Union.

Reply:
Done